aodndata/srs/srs_surface_waves.py

```diff
@@ -20,25 +21,30 @@
               "TOPEX"]
 
 
-def dest_path_srs_surface_waves(filepath):
-    platforms = '|'.join(VALID_SATS)
-
-    FILE_PATTERN = re.compile(r"""
-                                IMOS_SRS-Surface-Waves_MW_
-                                (?P<platform_code>{})_FV02_
-                                (?P<latitude>[0-9]{{3}})(S|N)-
-                                (?P<longitude>[0-9]{{3}})E-
-                                DM00\.nc$
-                                """.format(platforms), re.VERBOSE)
-
-    file_basename = os.path.basename(filepath)
-    if FILE_PATTERN.match(file_basename):
-        fields = get_pattern_subgroups_from_string(file_basename, FILE_PATTERN)
-    else:
-        raise InvalidFileNameError(
-            "file name: \"{filename}\" not matching regex to deduce dest_path".format(
-                filename=os.path.basename(filepath)))
-
-    return os.path.join(PREFIX_PATH, fields['platform_code'], file_basename)
-
-
+class SrsSurfaceWavesHandler(HandlerBase):
+    def __init__(self, *args, **kwargs):
+        super(SrsSurfaceWavesHandler, self).__init__(*args, **kwargs)
+        self.allowed_extensions = ['.nc', '.dir_manifest']
```

Comment:
I think what we'll want is to allow manifest files only.
The pipeline should watch a 'manifest' directory rather than the current incoming directory. The incoming directory stays for the user to be able to upload its data files but they don't trigger anything. Then when we upload a manifest in the 'manifest' directory it triggers the pipeline and the manifest points to the files in incoming.